lib/tower_bugsnag/bugsnag/event.ex

```diff
@@ -50,11 +50,13 @@ defmodule TowerBugsnag.Bugsnag.Event do
         plug_conn: plug_conn,
         metadata: metadata
       }) do
+    formatted_reason = Exception.format_exit(reason)
+
     %{
       exceptions: [
         %{
-          errorClass: "(exit) #{reason}",
-          message: reason,
+          errorClass: "(exit) #{formatted_reason}",
+          message: formatted_reason,
```

Comment:
Moved this to a separate more general PR https://github.com/mimiquate/tower_bugsnag/pull/19. This isn't specific to using `Bandit`.